src/test/java/com/google/api/codegen/discogapic/testdata/java/java_simplecompute.v1.json.baseline

```diff
@@ -8890,8 +8890,8 @@ public class AddressClient implements BackgroundResource {
   public final AggregatedListAddressesPagedResponse aggregatedListAddresses(ProjectName project) {
     AggregatedListAddressesHttpRequest request =
         AggregatedListAddressesHttpRequest.newBuilder()
-        .setProject(project == null ? null : project.toString())
-        .build();
+            .setProject(project == null ? null : project.toString())
```

Comment:
It is ok to keep it here, but please try to not mix unrelated refactoring/formatting with new features. For cases, when it happens, please mention it in the PR description.

Reply:
Edited the PR description.